### DISCUSSION TITLE
Increase beatmap difficulty limits up to 11

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#13205] [Ideas]
**Describe the new feature:** 
AR9 + DT = AR10.3
Playing with this AR is almost a separate skill, which is why simply AR 10 seems insufficient. Is it possible to give an opportunity to set for a map at the time of creation (not difficalty adjust) above 10?

**Comment by frenzibyte** [comment]
Not sure this will really happen given that any difficulty value beyond 10 is considered "extreme mode" here, but I guess this can stay as a proposal.

**Comment by smoogipoo** [comment]
From my experience, mappers don't jack up the AR for the sole purpose of increasing the skill requirement, and I really doubt they would if this was implemented. It's more commonly used to reduce density as a result of higher BPMs to more acceptable levels.

As I've mentioned in the past, AR>10 is really not a special skill by itself.

**Comment by AJT127** [comment]
honestly this is needed even in stable if that's even possible. maps are quickly becoming too much for AR10 to support their densities and being ranked anyways which is bad from a future-proofing standpoint

examples: https://osu.ppy.sh/beatmapsets/1263883#osu/2627024 https://osu.ppy.sh/beatmapsets/1201974#osu/2586420 https://osu.ppy.sh/beatmapsets/1390177#osu/2870806

and this is 1000000% definitely needed on lazer

**Comment by apollo-dw** [comment]
According to https://osu.ppy.sh/community/forums/posts/8193561, a "fresh take is required" with regards to mods for this. Is this something that anyone could provide input on? What were the issue with the proposals in the thread? There is a mapset stuck in purgatory waiting for this to be added.

**Reply by smoogipoo** [reply]
I don't know what "a fresh take on mods" means. Can you link the mapset?

It's quite hard to get AR>10 working for stable, because it actually isn't supported by game code. Stuff like pre-empt/fade-in times are actually hard-coded and break down at AR>10. In lazer, the DA mod can be used to set AR11, but it doesn't exactly replicate AR10+DT - it has the same fade _out_ times but I believe everything else including fade-in and followpoint fade times are only "close enough".

**Reply by Moltanicaa** [reply]
Maybe this? https://osu.ppy.sh/beatmapsets/1263883#osu/2669504

We need at least AR 10.3 for last diff 

A higher AR means fewer objects on the screen (in one frame, pattern), which is an important part of the game with more than 10 star.

**Reply by apollo-dw** [reply]
The linked forum post states "yes we've been confirmed with ppy that implementation should be quick work", but I'll take your word that stable AR>10 is significant work, and thus unlikely to be implemented. I really have my fingers crossed that lazer will allow map difficulty settings above the current stable maximum... Also yeah, Moltanicaa linked the map I was referring to.

**Comment by apollo-dw** [comment]
Assuming difficulty settings limits for maps are increased to 11, is it safe to assume that the "Extended Limits" behaviour in Difficulty Adjust would have to be adjusted appropriately? What limits would we be expecting here?
![image](https://user-images.githubusercontent.com/83023433/151798680-584ab0c0-2eda-4126-89b0-388e050baaca.png)

**Reply by mk56-spn** [reply]
> Assuming difficulty settings limits for maps are increased to 11, is it safe to assume that the "Extended Limits" behaviour in Difficulty Adjust would have to be adjusted appropriately? What limits would we be expecting here?
> ![image](https://user-images.githubusercontent.com/83023433/151798680-584ab0c0-2eda-4126-89b0-388e050baaca.png)
> 

Seems like a rather arbitrary assumption to think they'd change at all. Past 11 except for AR none of these settings really have been demonstrated to be of any use. HP is a mess but effectively hp 11 is SD after 1-2 misses on alot of maps

CS11 has yet to see any decent plays so same goes there

AR 11 and potentially OD11 are probably the only ones that might have any use if increased.

**Reply by apollo-dw** [reply]
The user experience sounds a little strange to me if you'd be able to set the difficulty settings to "extended limits" easily in the Editor, but you'd need to check something in Difficulty Adjust. I don't think that's arbitrary.

> CS11 has yet to see any decent plays

There is not enough opportunity for players to play with CS11 since there are no maps that were designed around it. I can guarantee that if CS11 became available for use in maps, it would have use. (as a player and a mapper, I am interested in this)

**Comment by Lyti4n** [comment]
Now that lazer only maps exist and are rankable it would be nice if this was revisited. This has been a highly requested feature for a long time specifically for AR and OD. I do agree with the argument that CS and HP should still be capped at 10 though.
